src/StaticArrays.jl

```diff
@@ -28,6 +28,7 @@ export @MVector, @MMatrix, @MArray
 
 export similar_type
 export push, pop, shift, unshift, insert, deleteat, setindex
+export drop_sdims
```

Comment:
Definitely.  I thought it might be useful, but we could make it internal until we figure out whether we really want it to be part of the public API.